### PR TITLE
Io plugins with surface mesh

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

Make the Io_plugins able to manage Surface_mesh_items as well as Polyhedron_items

This PR is based on #1954 
## Release Management

* Affected package(s):Polyhedron_demo
* Issue(s) solved (if any): fix #1931

### CODE REVIEW

**Comment by lrineau** [comment]
This PR is rather simple. it can be reviewed as-is.

It should be first rebased on the latest version of #1954.

File: Polyhedron/demo/Polyhedron/Plugins/IO/STL_io_plugin.cpp [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+            CGAL::Three::Scene_interface* scene_interface,
+            Messages_interface*) {
+    //get the references
+    this->scene = scene_interface;
+    this->mw = mainWindow;
+  }
+  QList<QAction*> actions() const {
+    return QList<QAction*>();
+  }
+  bool applicable(QAction*) const { return false;}
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Plugins/IO/STL_io_plugin.cpp]: Why is that I/O plugin turned into a regular plugin, with no action? Just to get the access to `this->mw` and then to the property?!

Comment (maxGimeno) [inline Polyhedron/demo/Polyhedron/Plugins/IO/STL_io_plugin.cpp]: Yes

**Comment by maxGimeno** [comment]
Ready for testing.

**Pull request merged**